lib/output.php

```diff
@@ -104,6 +104,18 @@ function genesis_sample_css() {
 		'
 	: '';
 
+	if ( ! is_customize_preview() ) {
+		$css .= ( has_custom_logo() ) ? sprintf(
+			'
+		.wp-custom-logo .site-container .custom-logo-link {
+			aspect-ratio: %1$s/%2$s;
```

Comment:
This styling appears, as expected:
<img width="896" alt="the-styling-is-applied" src="https://user-images.githubusercontent.com/4063887/111848732-6ccf4f00-88d1-11eb-94da-6e12d6a1d0ed.png">